typescript/infra/scripts/check-deploy.ts

```diff
@@ -15,30 +12,27 @@ async function check() {
   }
 
   const config = await getCoreEnvironmentConfig(environment);
-
-  const multiProvider = utils.initHardhatMultiProvider(config);
+  const [signer] = await ethers.getSigners();
+  const multiProvider = utils.initHardhatMultiProvider(config, signer);
 
   const core = AbacusCore.fromEnvironment(environment, multiProvider);
   const governance = AbacusGovernance.fromEnvironment(
     environment,
     multiProvider,
   );
 
-  const governors = await governance.governors();
+  const governor = await governance.governor();
 
   const governanceChecker = new AbacusGovernanceChecker(
     multiProvider,
     governance,
     config.governance,
   );
-  await governanceChecker.check(governors);
+  await governanceChecker.check(governor);
   governanceChecker.expectEmpty();
 
   const coreChecker = new AbacusCoreChecker(multiProvider, core, config.core);
-  await coreChecker.check(
-    governance.routerAddresses(),
-    coreEnvironments[environment],
-  );
+  await coreChecker.checkOwners(governance.routerAddresses());
```

Comment:
Should we also be calling `coreChecker.check()`?

Reply:
for `core`, this is the same